code/mobile/android/PhoneVR/build.gradle

```diff
@@ -31,6 +31,7 @@ allprojects {
     repositories {
         google()
         mavenCentral()
+        maven { url = uri("https://s01.oss.sonatype.org/content/repositories/snapshots") }
```

Comment:
Just this url seems a little odd. Hopefully its maintained officially.

Reply:
it's the official maven central snapshot repo: https://central.sonatype.org/publish/publish-maven/#performing-a-snapshot-deployment
and the author of zxing-cpp uploaded and gpg-signed that release

Reply:
Oh i see, Then there would be no security issues, Thanks !